lang/java/reef-bridge-client/src/main/java/org/apache/reef/bridge/client/YarnJobSubmissionClient.java

```diff
@@ -225,7 +225,7 @@ private void writeDriverHttpEndPoint(final File driverFolder,
     String trackingUri = null;
```

Comment:
It's probably worth it to put start and end messages that are on the INFO level.